### PR TITLE
Automatically upload Vulkan layer symbols.

### PR DESCRIPTION
**Pull request by florian-kuebler**

### CODE REVIEW

File: kokoro/builds/upload_symbols.sh [hunk 1]
```diff
@@ -72,6 +72,7 @@ function upload_symbol_file {
 function upload_debug_symbols {
   local api_key=$1
   local bin_folder=$2
```
Comment (IrinaShkviro) [inline kokoro/builds/upload_symbols.sh:74]: We also can change the second argument to the `build_folder`, and add '/bin' or '/lib' suffix below (when call upload_symbol_file function). But I don't have a strong opinion on this, so whatever you prefer.

**Review by IrinaShkviro — APPROVED** [review]
LGTM!

**Review by beckerhe — APPROVED** [review]
LGTM!

But I suggest we test this before merging. We could start a nightly build from this branch. The logfile should tell us whether it worked or not.

**Comment by florian-kuebler** [comment]
> LGTM!
> 
> But I suggest we test this before merging. We could start a nightly build from this branch. The logfile should tell us whether it worked or not.

Sounds good. Thanks!

File: kokoro/builds/upload_symbols.sh (hunk 1 above)
Comment (florian-kuebler) [reply kokoro/builds/upload_symbols.sh:74]: I thought about this. The main reason why I did it that way, was to break if I missed a usage of that function.

**Comment by florian-kuebler** [comment]
> > LGTM!
> > But I suggest we test this before merging. We could start a nightly build from this branch. The logfile should tell us whether it worked or not.
> 
> Sounds good. Thanks!

I've run the nightly collect build. It is green, but from the log file, I don't see any output related to symbol uploading.

**Pull request merged**